TargetRenamer/Trados.TargetRenamer/Trados.TargetRenamer/Converters/BooleanToVisibilityConditionalConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Data;
+
+namespace Trados.TargetRenamer.Converters
+{
+	public class BooleanToVisibilityConditionalConverter : IMultiValueConverter
```

Comment:
Do we need this converter?

Reply:
I'm using it to change the Visibility based on multiple conditions. For example, the delimiter's visibility if either suffix/prefix has been selected